### ISSUE TITLE
0.3.5 Composition Borders in 1.0.0

### ISSUE THREAD

**Issue by chrisdubya** [rveciana/d3-composite-projections#8]
I like the composition borders that you used in [this](http://bl.ocks.org/rveciana/170a76b8dc1f9cfd8b2d) example, but it seems that in 1.0.0 they come out in boxes like [this](http://bl.ocks.org/rveciana/5040be82aea528b6f785464f8816690f).

I'm using D3v4, so I'm assuming that 0.3.5 is incompatible, is there another way to achieve this type of border?

Thanks!

**Comment by rveciana** [comment]
You can still use this one: http://bl.ocks.org/rveciana/ee2119324e835e1bad42d0e4c1b9ab0d
The example you attached is the one that includes more territories that are not States but have Congressional Representatives.
What has changed is the projection for France.
An actually, the only thing added with the geoAlbersUsa are the paths with the borders for Alaska and Hawaii. The projection is the standard in D3.
So, basically, use geoAlbersUsa from the d3-composite-projections library.

**Comment by chrisdubya** [comment]
Ok, I see that.  I guess what I as asking is if it is possible to append those same paths for the borders for Alaska and Hawaii on the geoAlbersUsaTerritories projection territories.

**Comment by rveciana** [comment]
Ah, ok. The paths have these coordinates:
 var hawaii1 = lower48([-102.91, 26.3]);
    var hawaii2 = lower48([-104.0, 27.5]);
    var hawaii3 = lower48([-108.0, 29.1]);
    var hawaii4 = lower48([-110.0, 29.1]);

    var alaska1 = lower48([-110.0, 26.7]);
    var alaska2 = lower48([-112.8, 27.6]);
    var alaska3 = lower48([-114.3, 30.6]);
var alaska4 = lower48([-119.3, 30.1]);

You can add them as lines. But then the borders of the other regions won't appear.
I made them square to avoid so many lines with Samoa, Guam, etc. 
I think that the best option is drawing the desired borders "by hand" using the coordinates I gave and finding the ones you need for the other borders. 
If you need more indications, please ask.

**Comment by chrisdubya** [comment]
Ok, sounds good.  Thanks!
